packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -89,4 +89,14 @@ export const keys = {
   ENTER: 'Enter',
   ARROWUP: 'ArrowUp',
   ARROWDOWN: 'ArrowDown',
-};
\ No newline at end of file
+};
+
+export function getItemfromLocalStorage(key) {
+  const value = localStorage.getItem(key);
+  return value !== 'undefined' ? JSON.parse(value) : null;
```

Comment:
Do you mean `typeof(value) !== 'undefined'`?

Reply:
> If the value doesn' exist i thing `localStorage.getItem(key)` returns the string "undefined" which JSON.parse doesn't handle well.

No actually this was nathan's suggestion which I incorporated
I had previously done this:
`
return JSON.parse(localStorage.getItem(key))? JSON.parse(localStorage.getItem(key)) : null;`

Reply:
Acccording to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Storage/getItem#return_value), `localStorage.getItem` would return `null` when the key is not found.

[This example](https://jsfiddle.net/2cmzap1s/) confirms that `null` is returned when the key does not exist.

It is possible that the string `undefined` got stored by mistake?

Reply:
Ah hahaha that's possible. I'll check

Reply:
facepalm. Nice catch @lbergeron